src/Runner.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -1,4 +1,5 @@
 #if OS_WINDOWS
+#pragma warning disable CA1416
```

Comment:
Not sure why this pragma is needed. I didn't experience any problems on my branch with these warnings

Reply:
Taking it back. This is within `If Windows` block and I didn't test building on Windows